src/main-process/atom-application.js

```diff
@@ -421,10 +429,15 @@ class AtomApplication extends EventEmitter {
   // You can run the atom command multiple times, but after the first launch
   // the other launches will just pass their information to this server and then
   // close immediately.
-  listenForArgumentsFromNewProcess () {
-    if (!this.socketPath) return
+  async listenForArgumentsFromNewProcess (options) {
+    if (!options.test && !options.benchmark && !options.benchmarkTest) {
```

Comment:
I still wish we had a more succinct way to express "a regular editor window" in these options. (Out of scope for this PR though.)